parl/remote/tests/reset_job_test.py

```diff
@@ -62,22 +61,26 @@ class TestJob(unittest.TestCase):
     def tearDown(self):
         disconnect()
 
-    @timeout_decorator.timeout(seconds=300)
```

Comment:
Please do not remove this decroator as I remeber that it is used for the detection of a potential bug here.

Reply:
I find `ccc8f03` and `1768300` fail to pass because of this decorator in PY2.7 environment.

Maybe it's because we don't have ``timeout-decorator`` in the setup required packages.

Reply:
This explanation does not make sense. We have additional file for test environment setup(see https://github.com/PaddlePaddle/PARL/blob/develop/.teamcity/requirements.txt).

Reply:
Or it's maybe the timeout parameter is too small. The unittest stuck with `reset_job_test.py` on Friday, and pass until I remove the decorator.

Reply:
I see. Maybe we should add it back and change the timeout argument to 600s, and check if it can pass the test.

Reply:
It works.

Reply:
Great!